### ISSUE TITLE
SyntaxError: keyword argument repeated: type

### ISSUE THREAD

**Issue by sebastic** [eclipse/sumo#13650]
Installing the sumo Debian package fails to byte-compile the Python code:
```
 62s Setting up python3 (3.11.4-5) ...
 63s   File "/usr/share/sumo/tools/import/vissim/convert_detectors2SUMO.py", line 301
 63s     op.add_argument('--vissim-input', '-V', type=str, category="input", required=True, type=op.file,
 63s                                                                                        ^^^^^^^^^^^^
 63s SyntaxError: keyword argument repeated: type
 63s error running python rtupdate hook sumo-tools
 63s dpkg: error processing package python3 (--configure):
 63s  installed python3 package post-installation script subprocess returned error exit status 4
```
https://ci.debian.net/data/autopkgtest/testing/amd64/s/sumo/36079768/log.gz

This patch should resolve the issue:
```diff
--- a/tools/import/vissim/convert_detectors2SUMO.py
+++ b/tools/import/vissim/convert_detectors2SUMO.py
@@ -298,11 +298,11 @@ def get_conn_verb_rel(conn_tab, from_to_
 # MAIN
 if __name__ == '__main__':
     op = sumolib.options.ArgumentParser(description='detector conversion utility (VISSIM.inpx to SUMO)')
-    op.add_argument('--vissim-input', '-V', type=str, category="input", required=True, type=op.file,
+    op.add_argument('--vissim-input', '-V', category="input", required=True, type=op.file,
                     help='VISSIM inpx file path')
-    op.add_argument('--output-file', '-o', type=str, category="output", required=True, type=op.file,
+    op.add_argument('--output-file', '-o', category="output", required=True, type=op.file,
                     help='output file name')
-    op.add_argument('--SUMO-net', '-S', type=str, category="input", required=True, type=op.net_file,
+    op.add_argument('--SUMO-net', '-S', category="input", required=True, type=op.net_file,
                     help='SUMO net file path')
     args = op.parse_args()
     print("\n", args, "\n")
--- a/tools/import/vissim/tls_vissimXML2SUMOnet_update.py
+++ b/tools/import/vissim/tls_vissimXML2SUMOnet_update.py
@@ -527,7 +527,7 @@ if __name__ == '__main__':
     op = sumolib.options.ArgumentParser(
         description='TLS conversion utility (VISSIM.inpx to SUMO)')
     op.add_argument('--vissim-input', '-V',
-                    type=str, category="input", required=True, type=op.file,
+                    category="input", required=True, type=op.file,
                     help='VISSIM inpx file path')
     op.add_argument('--SUMO-net', '-S',
                     category="input", required=True, type=op.net_file,
```

**SUMO-version:** 1.18.0

**Operating System:** Debian GNU/Linux trixie/sid amd64

**Comment by m-kro** [comment]
Solved via a123537

**Comment by m-kro** [comment]
@sebastic Thanks!
